week-2-wide-perspective/w02e05-simple-server-side/src/users/users.controller.ts

```diff
@@ -0,0 +1,24 @@
+import { log } from 'console';
+import { Request, Response, Router } from 'express';
+import { logger } from '../logger';
+import { validationMiddleware } from '../middleware/validation.middleware';
+import { getUserToken } from './user.service';
+import { UserLoginScheme } from './users.validator';
+
+
+export const usersController = Router();
+
+usersController.post("/", [validationMiddleware({ body: UserLoginScheme })], async (req: Request, res: Response, next: any) => {
+    const { email, password } = req.body
+
+    logger.log("info", `Try to get token for:  login: ${email} password:${password}`)
+
+    try {
+      const user = await getUserToken(email, password);
+      res.json(user);
+    } catch (error) {
```

Comment:
Jak doinstalujesz liba o którym wspominałem wyżej do `async-errors` to te `try...catch`e Ci się mocno uproszczą bo nie trzeba ich będzie pisać.